### DISCUSSION TITLE
Review commit by commit

### DISCUSSION THREAD

**Discussion by arvid123** [community/community#54887] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Feature request: ability to go through a request commit by commit in chronological order instead of just file by file. I think this could reduce the overwhelming effects felt when reviewing a large pull request.

**Comment by Startouf** [comment]
Is it me or is this something that has become available recently ? I mean now I seem to be able to review a specific commit (and the action of having to go through all commits remain a manual action)
